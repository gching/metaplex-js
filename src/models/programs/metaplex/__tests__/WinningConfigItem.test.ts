import { WinningConfigItem, WinningConfigType } from '../WinningConfigItem';

describe('WinningConfigItem', () => {
  it('should instantiate with the correct values', () => {
    const inst = new WinningConfigItem({
      amount: 0,
      safetyDepositBoxIndex: 0,
      winningConfigType: WinningConfigType.TokenOnlyTransfer,
    });

    expect(inst).toMatchSnapshot();
  });
});
