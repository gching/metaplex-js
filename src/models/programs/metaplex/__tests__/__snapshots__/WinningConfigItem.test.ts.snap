// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WinningConfigItem should instantiate with the correct values 1`] = `
WinningConfigItem {
  "amount": 0,
  "safetyDepositBoxIndex": 0,
  "winningConfigType": 0,
}
`;
